galleon-plugins/src/main/java/org/wildfly/galleon/plugin/WfInstallPlugin.java

```diff
@@ -564,15 +564,15 @@ private void provisionExampleConfigs() throws ProvisioningException {
 
         final Path examplesTmp = runtime.getTmpPath("example-configs");
         final ProvisioningLayoutFactory factory = runtime.getLayout().getFactory();
-        final ProgressTracker<String> examplesTracker = factory.getProgressTracker("JBEXAMPLES");
+        final ProgressTracker<List<Object>> examplesTracker = factory.getProgressTracker("JBEXTRACONFIGS");
         final List<String> trackedPhases = List.of(ProvisioningLayoutFactory.TRACK_LAYOUT_BUILD, ProvisioningLayoutFactory.TRACK_PACKAGES,
                 TRACK_MODULES_BUILD, ProvisioningLayoutFactory.TRACK_CONFIGS);
-        examplesTracker.starting(trackedPhases.size() + 1);
```

Comment:
@jfdenise shouldn't we keep the `starting` call? Otherwise the tracker can't provide a completion percentage

Reply:
I am not sure percentage makes sense there. We are printing the name of the generated example configs. As we do for the default configs.

Reply:
I see what you mean, but we know how many "phases" the generation needs to go through, so we can use that even if they don't represent the count of generated examples. At very least it'll give some idea about how for in the phase we are